doc/source/changes/version_0_28.rst.inc

```diff
@@ -255,8 +255,20 @@ Miscellaneous improvements
 
   Closes :issue:`488`.
 
-* deprecated `nan_equal` function in favor of `equal` function which has the same optional arguments
-  as the `LArray.equals` method (closes :issue:`593`).
+* deprecated `nan_equal` function in favor of `equals` function. The `equals` function has the same optional
```

Comment:
The current situation with both an equals **function** and an equals **method** which do different things seems very confusing to me. One way to solve this would be to make the equals function private, rename LArray.equals into something like LArray.allequal and add a new method LArray.equals which does what the equals function does. There are other options to fix this though. We might need to rename the corresponding session functions.

Reply:
Once again, I am pretty sure this function will only be used in larray-project. 
IMO, like pandas.DataFrame.equals, LArray.equals should return a boolean value.
I can rename the function 'equals' as `element_equals`?

Reply:
yes, that's a good option too.